docs/guides/index.md

```diff
@@ -527,15 +527,16 @@ Events][animatingonevents] below.
 
 In JavaScript, we grab the box with [`querySelector`][queryselector], use
 [`addEventListener`][addeventlistener], and then [`setAttribute`][setattribute]
-to make the box grow its scale when its hovered over. Note that A-Frame adds
+to make the box grow its scale when its hovered over. The code must appear after
+the scene so that the box has loaded when the script runs. Note that A-Frame adds
 features to `setAttribute` to work with multi-property components. We can pass
 a full `{x, y, z}` object as the second argument.
 
 ```js
 <script>
-  var boxEl = document.querySelector('a-box')
+  var boxEl = document.querySelector('a-box');
```

Comment:
I think we have to close this PR because I deleted that fork when trying to get a fresh copy of the repo. I'm still a novice with regards to branching, forking etc. I can make another PR.